apps/extension/src/apis/axios.ts

```diff
@@ -0,0 +1,68 @@
+import apiRequest from "./axiosInstance";
+export interface PostArticleRequest {
+  url: string;
+  categoryId: number;
+  memo?: string | null;
+  remindTime?: string | null; 
+}
+
+export const postArticle = async (data: PostArticleRequest) => {
+  const response = await apiRequest.post("/api/v1/articles", data);
+  return response.data;
+};
+
+
+export interface postSignupRequest {
+  email: string;
+  remindDefault: string
+  fcmToken: string;
+}
+
+export const postSignup = async (data: postSignupRequest) => {
+  const response = await apiRequest.post("/api/v1/auth/signup", data);
+  return response.data;
+};
+
+export const getCategoriesExtension = async () => {
+  const response = await apiRequest.get("/api/v1/categories/extension");
+  return response.data;
+};
```

Comment:
```suggestion
export const getCategoriesExtension = async () => {
  const { data }  = await apiRequest.get("/api/v1/categories/extension");
  return data;
};
```

바로 구조 분해 할당도 가능합니다!!